loop-templates/templates/src/aptitudes/index.ts.squirrelly

```diff
@@ -46,6 +49,9 @@ export {
 {{ @if (it.aptitudes.includes('process')) }}
   processExample,
 {{ /if }}
+{{ @if (it.aptitudes.includes('search')) }}
+  runSearchExample,
```

Comment:
The naming convention here is different than the rest so it seems out of place. Can we change it to `searchExample` and `searchExample.run()`?